openapi/interzoid.weatherzip/types.bal

```diff
@@ -27,4 +27,9 @@ public type WeatherData record {
     string Weather?;
     string WindDir?;
     string WindMPH?;
+    string AirQuality?;
+    string AirQualityIndex?;
+    string AirQualityCode?;
+    string Sunrise?;
+    string Sunset?;
```

Comment:
We should add these in the openapi.yaml as well.

Otherwise in the future if we re-generate the connector, we'll miss these fields

Reply:
Added to `Openapi.yml`.